week07/my-todo-list/src/pages/HomePage.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from "react";
+import "../App.css";
+import CustomDate from "../CustomDate";
+
+const HomePage = () => {
+  const quote_url = "https://api.quotable.io/random";
+  const [quote, setQuote] = useState("");
+  const [author, setAuthor] = useState("");
+
```

Comment:
- [x] 3. 로딩 시간 - 명언 api
```suggestion
const [isLoading, setIsLoading] = useState(true);
```